Directory.Packages.props

```diff
@@ -86,78 +86,78 @@
     <PackageVersion Include="DnsClient" Version="1.7.0" />
-    <PackageVersion Include="Google.Protobuf" Version="3.24.0" />
-    <PackageVersion Include="Grpc.AspNetCore" Version="2.60.0" />
-    <PackageVersion Include="Grpc.Net.ClientFactory" Version="2.60.0" />
-    <PackageVersion Include="Grpc.Tools" Version="2.62.0" />
+    <PackageVersion Include="Google.Protobuf" Version="3.27.1" />
+    <PackageVersion Include="Grpc.AspNetCore" Version="2.63.0" />
+    <PackageVersion Include="Grpc.Net.ClientFactory" Version="2.63.0" />
+    <PackageVersion Include="Grpc.Tools" Version="2.64.0" />
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
-    <PackageVersion Include="KubernetesClient" Version="13.0.11" />
-    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.0" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.6.1" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.6.1" />
-    <PackageVersion Include="MongoDB.Driver" Version="2.25.0" />
+    <PackageVersion Include="KubernetesClient" Version="14.0.2" />
+    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.1" />
+    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.8.1" />
+    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.8.0" />
+    <PackageVersion Include="MongoDB.Driver" Version="2.26.0" />
     <PackageVersion Include="MongoDB.Driver.Core.Extensions.DiagnosticSources" Version="1.4.0" />
     <PackageVersion Include="MySqlConnector.DependencyInjection" Version="2.3.6" />
     <PackageVersion Include="MySqlConnector.Logging.Microsoft.Extensions.Logging" Version="2.1.0" />
-    <PackageVersion Include="NATS.Net" Version="2.2.1" />
-    <PackageVersion Include="Npgsql.DependencyInjection" Version="8.0.2" />
-    <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.2" />
-    <PackageVersion Include="Oracle.EntityFrameworkCore" Version="8.21.140" />
-    <PackageVersion Include="Polly.Core" Version="8.3.1" />
-    <PackageVersion Include="Polly.Extensions" Version="8.3.1" />
+    <PackageVersion Include="NATS.Net" Version="2.2.3" />
+    <PackageVersion Include="Npgsql.DependencyInjection" Version="8.0.3" />
+    <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.4" />
+    <PackageVersion Include="Oracle.EntityFrameworkCore" Version="8.23.40" />
+    <PackageVersion Include="Polly.Core" Version="8.4.0" />
+    <PackageVersion Include="Polly.Extensions" Version="8.4.0" />
     <PackageVersion Include="Pomelo.EntityFrameworkCore.MySql" Version="8.0.2" />
     <PackageVersion Include="Qdrant.Client" Version="1.9.0" />
     <PackageVersion Include="RabbitMQ.Client" Version="[6.8.1,7.0.0)" />
-    <PackageVersion Include="StackExchange.Redis" Version="2.7.33" />
+    <PackageVersion Include="StackExchange.Redis" Version="2.8.0" />
```

Comment:
It's kind of surprising to jump out in front of `main` with our dependencies. We haven't taken this update there. Do we plan on making the same change in `main`?

Reply:
Yeah it seems unwise to bump deps for the first time in a servicing branch and then ship very quickly from there, rather than updating them in main and allowing some bake time. That said, we should be regularly bumping deps in main (we discussed weekly IIRC) so this becomes less risky.

Reply:
Yes, this commit will get merged into main after releasing as part of the release/8.0 -> main merge that happens after each release

Reply:
> that happens after each release

That doesn't seem like the right process/steps/flow. I think these should happen in main first to get more coverage.

Reply:
Sounds good to me. I can revert this particular one to match what we have in main.

Reply:
To be honest, instead of reverting, I think we should do it in both now. Getting the updated dependency in our 8.0.2 release sounds good to me. I just want us to also make it in main at the same time (or sooner).

Reply:
Given 8.0.2 is set to ship this week, and we don't have that much time, I think I like your original suggestion of keeping with the versions we have in main now to reduce risk as people have been validating main now for a while. I'll update this PR to use versions that match main, and then I'll put up a PR in main for the newer versions so we can ship those in 8.1. Sound good?

Reply:
That sounds good, with the addition of ensuring we include any updates that lift transitive dependencies with vulnerabilities (e.g. `Azure.Identity`).